utilities/test_suite/rpp_test_suite_audio.h

```diff
@@ -37,6 +37,14 @@ std::map<int, string> audioAugmentationMap =
     {0, "non_silent_region_detection"},
 };
 
+// Golden outputs for Non Silent Region Detection
+std::map<string, std::vector<int>> NonSilentRegionReferenceOutputs =
```

Comment:
How many outputs will be part of header file? Okay for now, but we should probably move it to a better location once other audio functions start adding up